instrumentation/sinatra/example/Gemfile

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 source 'https://rubygems.org'
-
+gem 'puma'
```

Comment:
Does webrick not work by default?

Reply:
Yeah I guess I am surprised that it does not work and it should probably use whatever is bundled in the Ruby stdlib, which is what I thought sinatra used by default? but I guess not. 

Puma should be fine.